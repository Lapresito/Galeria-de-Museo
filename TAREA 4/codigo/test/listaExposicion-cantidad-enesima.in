crearListaExposiciones
cantidadExposicionesListaExposiciones

crearExposicion 02/03/2022 01/05/2022 10 3
agregarAExposicion 9 0 1 2 3 4 5 6 7 8 
agregarAListaExposiciones
obtenerNesimaExposicionListaExposiciones 1

crearExposicion 01/01/2022 01/07/2022 10 1
agregarAExposicion 9 0 1 2 3 4 5 6 7 8 
agregarAListaExposiciones
cantidadExposicionesListaExposiciones
obtenerNesimaExposicionListaExposiciones 2
obtenerNesimaExposicionListaExposiciones 1

crearExposicion 01/02/2022 01/06/2022 10 2
agregarAExposicion 9 0 1 2 3 4 5 6 7 8 
agregarAListaExposiciones
cantidadExposicionesListaExposiciones
obtenerNesimaExposicionListaExposiciones 1
obtenerNesimaExposicionListaExposiciones 2
obtenerNesimaExposicionListaExposiciones 3

liberarListaExposiciones
Fin