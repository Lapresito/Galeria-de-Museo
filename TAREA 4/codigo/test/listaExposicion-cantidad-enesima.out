Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Lista de exposiciones creada con exito.
2> Hay 0 exposiciones en la lista de exposiciones.
3> Exposicion creada con exito.
4> Piezas agregadas a la exposicion de forma exitosa.
5> Se ha agregado la exposicion a la lista de forma exitosa.
6> Exposicion en posicion 1 de la lista:
Exposicion #3 del 2/3/2022 al 1/5/2022
Piezas: 0 1 2 3 4 5 6 7 8 
7> Exposicion creada con exito.
8> Piezas agregadas a la exposicion de forma exitosa.
9> Se ha agregado la exposicion a la lista de forma exitosa.
10> Hay 2 exposiciones en la lista de exposiciones.
11> Exposicion en posicion 2 de la lista:
Exposicion #3 del 2/3/2022 al 1/5/2022
Piezas: 0 1 2 3 4 5 6 7 8 
12> Exposicion en posicion 1 de la lista:
Exposicion #1 del 1/1/2022 al 1/7/2022
Piezas: 0 1 2 3 4 5 6 7 8 
13> Exposicion creada con exito.
14> Piezas agregadas a la exposicion de forma exitosa.
15> Se ha agregado la exposicion a la lista de forma exitosa.
16> Hay 3 exposiciones en la lista de exposiciones.
17> Exposicion en posicion 1 de la lista:
Exposicion #1 del 1/1/2022 al 1/7/2022
Piezas: 0 1 2 3 4 5 6 7 8 
18> Exposicion en posicion 2 de la lista:
Exposicion #2 del 1/2/2022 al 1/6/2022
Piezas: 0 1 2 3 4 5 6 7 8 
19> Exposicion en posicion 3 de la lista:
Exposicion #3 del 2/3/2022 al 1/5/2022
Piezas: 0 1 2 3 4 5 6 7 8 
20> Lista exposiciones liberada con exito.
21> Fin.
